pkgs/development/libraries/linenoise/default.nix

```diff
@@ -0,0 +1,30 @@
+  { stdenv, fetchFromGitHub, lib }:
+
+  stdenv.mkDerivation rec {
+    name = "linenoise-${version}";
+    version = "1.0.10";  # Its version 1.0 plus 10 commits
+
+    src = fetchFromGitHub {
+      owner = "antirez";
+      repo = "linenoise";
+      rev = "c894b9e59f02203dbe4e2be657572cf88c4230c3";
```

Comment:
The thing is that I kind of invented the version number. The only tag on the repo is "1.0", but it's 10 commits away of `head`, hence the `1.0.10` version.I hope it's ok with you?